styleguide/_themes/derek/scss/components/ticker.scss

```diff
@@ -0,0 +1,90 @@
+@import '../../../global/scss/swatch';
+@import '../../../global/scss/fonts';
+@import '../../../global/scss/bootstrap_variables';
+@import '../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/clearfix';
+@import '../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/grid';
+
+$ticker-span-bg: rgba(200, 0, 0, .5);
+
+.ticker {
+  background-color: $rackspace-red;
+  width: 100%;
+}
+
+.ticker-container {
+  @include container-fixed;
+  max-width: 960px;
+
+  @media (min-width: $screen-sm-min) {
+    width: $container-sm;
+  }
+  @media (min-width: $screen-md-min) {
+    width: $container-md;
+  }
+  @media (min-width: $screen-lg-min) {
+    width: $container-lg;
+  }
+}
+
+.ticker-row {
+  @include make-row;
+}
+
+.ticker-label {
+  @include make-md-column(4);
+  color: $white;
+  font-family: $subhead;
+  padding: 30px 0;
+  text-align: center;
+  text-shadow: 1px 2px $black;
+  text-transform: uppercase;
+}
+
+.ticker-countdown-column {
+  @include make-md-column(4);
+  padding: 0;
+}
+
+.ticker-countdown {
+  background-color: $white;
```

Comment:
in sass you should only extend placeholder, I learned that after I taught you to do this :/